src/lib/components/input/input.js

```diff
@@ -1,83 +1,98 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-import { componentPropTypes, defaultComponentPropTypes } from '../../utils/componentPropTypes';
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  componentPropTypes,
+  defaultComponentPropTypes,
+} from "../../utils/componentPropTypes";
+import { handleInputChange } from "../oldInput/utils.ignore";
```

Comment:
I don't like to have this oldInput thing here. Maybe it is something that we can add to this new input component if we need it

Reply:
Yeah for sure. The idea is to only use the `oldInput` files for the immediate future. Once we're sure the refactored `input` is working, we will delete the `oldInput` folder

Reply:
I've copied in the `utils.ignore.js` file from `oldInput` folder into `input` folder and have adjusted the import call to suit... I hope this is ok @rin and @davidpaley - https://github.com/aimementoring/blueprint/pull/179/commits/b498d9454b5c4d19e8c1d346a43bb92eb47c37a5